reporters/reporter-default/src/main/java/org/radargun/reporting/csv/CsvReporter.java

```diff
@@ -66,7 +83,7 @@ private void reportTest(Report report, Report.Test test) {
          fileWriter = prepareOutputFile(report, test.name, "");
          int it = 0;
          Set<String> columns = new TreeSet<String>();
-         ArrayList<Map<String, String>> rows = new ArrayList();
+         ArrayList<Map<String, String>> rows = new ArrayList<Map<String, String>>();
```

Comment:
Yes. Using just ArrayList() produces a warning. Another way to fix this is the diamond operator:
ArrayList<Map<String, String>> rows = new ArrayList<>();